homeassistant/components/plex/config_flow.py

```diff
@@ -214,3 +223,42 @@ def __init__(self):
         """Import from Plex configuration."""
         _LOGGER.debug("Imported Plex configuration")
         return await self.async_step_server_validate(import_config)
+
+
+class PlexOptionsFlowHandler(config_entries.OptionsFlow):
+    """Handle Plex options."""
+
+    def __init__(self, config_entry):
+        """Initialize Plex options flow."""
+        self.options = dict(config_entry.options)
```

Comment:
This won't work since you have a multi level dict you will change config_entry.options[MP_DOMAIN] directly when changing self.options[MP_DOMAIN]